### ISSUE TITLE
Question:clarify NVPTX status?

### ISSUE THREAD

**Issue by oscarbg** [brechtsanders/winlibs_mingw#43]
Hi,
seems some NVPTX support is there but also:
"Offloading support for nvptx (nVidia GPU) is included, but GCC was not built with --enable-offload-targets=nvptx-none"
trying something like:
`gfortran -Wall -O2 -g -fopenmp -foffload=nvptx-none -c kernelomp.f90 `
shows error:
`gfortran: fatal error: GCC is not configured to support nvptx-none as offload target`
so how can test NVPTX OMP offload currently?
thanks..
using winlibs-x86_64-posix-seh-gcc-10.2.0-mingw-w64-8.0.0-r6b

**Comment by brechtsanders** [comment]
Hi,
As far as the winlibs build is concerned this still experimental. I'm not enough of an expert in offloading to test this.
I will try to build gcc in the next release with `--enable-offload-targets=nvptx-none`.
It would be great if you can test it then.
I will keep this ticket open to remind me.

**Comment by oscarbg** [comment]
yes I’m willing to test and report as soon as a build is posted..
thanks

**Comment by brechtsanders** [comment]
Can you please test again with the latest release from https://github.com/brechtsanders/winlibs_mingw/releases/tag/10.2.0-11.0.0-8.0.0-r8 ?

**Comment by brechtsanders** [comment]
Update: GCC 11 was released but currently doesn't build nvptx properly (see [my bug report](https://gcc.gnu.org/bugzilla/show_bug.cgi?id=100293))

**Comment by brechtsanders** [comment]
Do you still have issues with https://github.com/brechtsanders/winlibs_mingw/releases/tag/10.3.0-12.0.0-9.0.0-r2
 ?

**Comment by oscarbg** [comment]
sorry for not answering will try to test soon..
